Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -323,7 +324,7 @@ def longFileLinter : Linter where run := withSetOptionIn fun stx ↦ do
   let linterBound := linter.style.longFile.get (← getOptions)
   if linterBound == 0 then
     return
-  let defValue := linter.style.longFile.defValue
+  let defValue := 1500
```

Comment:
Doesn't this hardcode the value of `1500` ignoring the option?

Reply:
1. As you say, this does not hard-code the linter option, since otherwise CI would have failed on long files. It *does* hard-code what the linter considers to be the expected length of a file. `defValue` is used to inform the user that they should not use `set_option linter.style.longFile x` with `x ≤ 1500`. It would be nice to be able to retrieve the value of the `weak` option set in the lake-file, but I am doubtful of whether that is accessible or not.
2. I did test downstream and the linter is indeed not flagging long files in the test-project that I used.

Reply:
Maybe as a further clarification, the `defValue` that you flagged is *not* `linter.style.longFile.defValue`: the former is 1500, the latter is 0.